client/Renderer.ts

```diff
@@ -0,0 +1,18 @@
+// Client-side JavaScript goes here!
+
+import Requester from "./Requester";
+
+const requester = new Requester();
+
+const exampleButton = document.getElementById("example-button") as HTMLElement;
+const exampleResponseContainer = document.getElementById(
+  "example-response-container"
+) as HTMLElement;
+
+exampleButton.addEventListener("click", async () => {
+  const response = await requester.request<string>(
+    "example-channel",
+    "optional argument goes here"
+  );
+  exampleResponseContainer.innerHTML = `The following is being retrieved from the backend via the example channel:<br><br>${response}`;

```

Comment:
Is there some way I could load a `<script>` tag into this? That would be a security hole.